source/features/easy-toggle-files.tsx

```diff
@@ -15,14 +15,43 @@ function toggleFile(event: DelegateEvent<MouseEvent>): void {
 	}
 }
 
+function toggleCSFile(event: DelegateEvent<MouseEvent>): void {
+	const elementClicked = event.target as HTMLElement;
+	const headerBar = event.delegateTarget;
+
+	// The clicked element is either the bar itself or one of its children excluding the button
+	if (elementClicked === headerBar || (elementClicked.parentElement === headerBar && elementClicked.tagName !== 'BUTTON')) {
+		$(['[aria-label^="Collapse "]', '[aria-label^="Expand "]'], headerBar)!
+			.dispatchEvent(new MouseEvent('click', {bubbles: true, altKey: event.altKey}));
+	}
+}
+
 function init(signal: AbortSignal): void {
 	delegate('.file-header', 'click', toggleFile, {signal});
 }
 
+function initCS(signal: AbortSignal): void {
+	delegate('div[data-testid="results-list"] > div > div:first-child', 'click', toggleCSFile, {signal});
```

Comment:
This selector is too fickle. Maybe:

```suggestion
	delegate('div:has(>:is([aria-label^="Collapse "], [aria-label^="Expand "]))', 'click', toggleCSFile, {signal});
```
Probably we should also extract this selector to `selectors.ts` and add a test for it.